recipes/configs/qwen2/0.5B_lora_single_device.yaml

```diff
@@ -39,20 +38,22 @@ checkpointer:
   recipe_checkpoint: null
   output_dir: /tmp/Qwen2-0.5B-Instruct-lora-finetune
   model_type: QWEN2
+
 resume_from_checkpoint: False
 
 # Dataset and Sampler
 dataset:
   _component_: torchtune.datasets.alpaca_cleaned_dataset
 seed: null
 shuffle: True
-batch_size: 2
+batch_size: 4
 
 # Optimizer and Scheduler
 optimizer:
   _component_: torch.optim.AdamW
   weight_decay: 0.01
-  lr: 3e-4
+  lr: 2e-3
```

Comment:
is the change in LR is because we tested it / saw it somewhere, or is it just a guess?

Reply:
I tested when benchmarking